contracts/modules/andromeda-address-list/src/contract.rs

```diff
@@ -77,17 +81,19 @@ pub fn execute(
 
 pub fn handle_execute(ctx: ExecuteContext, msg: ExecuteMsg) -> Result<Response, ContractError> {
     match msg {
-        ExecuteMsg::AddActorPermission { actor, permission } => {
-            execute_add_actor_permission(ctx, actor, permission)
+        ExecuteMsg::AddActorPermission { actors, permission } => {
```

Comment:
I wonder if there's a better name for this? `PermissionActors` maybe?

Reply:
Yeah I think that's a better name. We need to change `RemoveActorPermission` as well. `RemovePermissionActors` ?

Reply:
Maybe just `RemovePermissions`?